src/mono/mono/mini/unwind.c

```diff
@@ -1167,8 +1166,7 @@ mono_unwind_decode_fde (guint8 *fde, guint32 *out_len, guint32 *code_len, MonoJi
 	while (p < cie + cie_len + 4) {
 		if (*p == DW_CFA_nop)
```

Comment:
Should this condition be inverted and hoisted into the `while`, like what was done at line 1259?

Reply:
No